app/assets/javascripts/discourse/app/templates/topic-list-header-column.hbr

```diff
@@ -13,7 +13,7 @@
       </span>
     {{/if ~}}
   {{/if ~}}
-  {{view.localizedName}}
+  <span>{{view.localizedName}}</span>
```

Comment:
Very minor, but I wonder whether this change is necessary. If it is, fine to keep, but if it's not necessary, might best to leave this as is (doesn't seem relevant to the other changes in this PR and can cause regressions with custom themes).